client/src/redux/actions/authActions.js

```diff
@@ -55,24 +55,19 @@ export const register =
 // Login User
 export const login = (email, password) => async (dispatch) => {
   const body = JSON.stringify({ email, password })
-
```

Comment:
keep this in here, I tend to opt for more spaces but that's just personal preference.

Reply:
gotcha. yeah me too, shall keep in mind moving forward